node-src/tasks/build.ts

```diff
@@ -87,15 +87,12 @@ export const buildStorybook = async (ctx: Context) => {
     // and it failed, that means we couldn't find it. This probably means they haven't
     // installed the right dependency or run from the right directory
     if (
-      ctx.options.inAction &&
-      (ctx.options.playwright || ctx.options.cypress) &&
-      e.message.match(e2EbuildBinName)
+      e.message.match(e2EbuildBinName) &&
```

Comment:
Not sure if we even need this check on the bin name. The command that failed is in the error message, so I don't know when this wouldn't match for E2E projects.

Reply:
Right, I guess it's probably rendundant.